dbms/src/Functions/FunctionsStringRegex.cpp

```diff
@@ -306,8 +306,8 @@ struct MultiMatchAnyImpl
         MultiRegexps::ScratchPtr smart_scratch(scratch);
 
         auto on_match = []([[maybe_unused]] unsigned int id,
-                           unsigned long long /* from */,
-                           unsigned long long /* to */,
+                           unsigned long long /* from */, // NOLINT
+                           unsigned long long /* to */, // NOLINT
```

Comment:
Why we do so?

Reply:
It's the interface of Hyperscan.

And `long long` is not the synonim of `uint64_t`.

uint64_t can correspond to either long and long long.
long and long long have the same behaviour on Linux amd64 and Mac OS amd64 and FreeBSD amd64 but not on Windows amd64.